### DISCUSSION TITLE
Does the design of lazer beatmap selector too dense ?

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#14804] [Q&A]
Hey,
I don't know if its the good place to debate about that, but I want to tell my experience about lazer.
Actually lazer is pretty good for stability and gameplay and im really hypped about the future of lazer, but I didn't like the actual design in the beatmap selector because for me it contain too much informations. 
I know that a lot of peoples love that but I found that the beatmap selector of stable is more clean because you have less data and you can watch the background of the selected map without having a lot of text on it. 
Frenquently im talking about lazer new releases with some friends and most of them don't really want to switch to lazer even if it will comming out because of the design. They just wants to have a simple interface with understable visual of the actual map and I can't contradict them because im really not loving the actual beatmap menu selector.

I know that a redesign of the game is still in work but is it planned to have something like an option to enjoy the background of the selected beatmap or something that clean the most of text that we don't really want  ? Maybe a compact version of the actual design.

stable
![image](https://user-images.githubusercontent.com/50368878/133935260-8803cf0d-4bdf-4cd8-a364-a945b07bf4aa.png)

lazer
![image](https://user-images.githubusercontent.com/50368878/133935330-82ca11f4-0a4f-404d-b22c-d8e55d09bdca.png)

**Comment by bdach** [comment]
there is a pending redesign of the song select screen, the work-in-progress version can be previewed at https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1. it may ease some of your concerns.

**Reply by Biokax** [reply]
It seems definitively better but it still hide a large part of the background, I love minimalist design and I think it stil too much big on the screen, and it doesn't have transparency

**Reply by bdach** [reply]
there isn't one perfect solution. even with the new beatmap cards (the ones visible on web) users were reporting feedback that there is both too much and not enough info shown. heavily depends on who is giving the feedback.

**Reply by Biokax** [reply]
maybe making users choose between classic version and compact minimalist design can make most of players happy with. 
And skinning menu for theses who don't love both 🤔

**Reply by peppy** [reply]
You will be able to change the layout of your song select screen. Please see what you can do in gameplay using the skin layout editor and expect the same thing to be available at song select.

**Reply by FazlyMR** [reply]
I'm curious how the left-side would scale for non-16:9 screens (such as 4:3, or 5:3)...
